_common

```diff
@@ -215,7 +215,7 @@ $body
 
 latest_published_tw_builds() {
     local latest_builds group_id=$1
-    latest_builds=("$(runcurl -s "$tw_openqa_host/group_overview/$group_id.json" \
+    latest_builds=("$(runcurl -s -S "$tw_openqa_host/group_overview/$group_id.json" \
```

Comment:
I commonly see (and use) `curl -sS` so I would use the shorter form here as well